### PR TITLE
update the code so that it can take 'vdw_corr' as input parameter 

### PR DESCRIPTION
**Pull request by sandipde**
The code has been updated with the option to take vdw_corr as input parameter in the __init__ class. the write_input functin has been updated accordingly. In addition a possible bug which would have arised in connection with 'exxdiv_treatment' parameter and now with vdw_corr has been fixed.

### CODE REVIEW

File: espresso/espresso.py [hunk 1]
```diff
@@ -1516,11 +1517,11 @@ def write_input(self, inputname='pw.inp', calculation=None,
             if value is not None:
                 print('  {0:s}={1:s},'.format(attr, num2str(value)), file=finp)
 
-        sys_str_attrs = ['exxdiv_treatment']
+        sys_str_attrs = ['exxdiv_treatment','vdw_corr']
         for attr in sys_str_attrs:
             value = getattr(self, attr)
             if value is not None:
-                print('  {0:s}={1:s},'.format(attr, value), file=finp)
+                print('  {0:s}=\'{1:s}\','.format(attr, value), file=finp)
```
Comment (lmmentel) [inline espresso/espresso.py]: how about updating this line to ` print("  {0:s}='{1:s}',".format(attr, value), file=finp)`

**Review by lmmentel — CHANGES_REQUESTED** [review]

**Comment by lmmentel** [comment]
Did you manage to run ase-espresso with `vdw_corr` set to check if it actually works?

File: espresso/espresso.py (hunk 1 above)
Comment (sandipde) [reply espresso/espresso.py]: I am okay  with both.  :-)

**Comment by sandipde** [comment]
> Did you manage to run ase-espresso with `vdw_corr` set to check if it actually works?

I did not run the espresso yet (our cluster is down until tomorrow) but checked the input file creation and everything should work.

**Comment by sandipde** [comment]
@lmmentel  incorporated the suggestion. please review.

File: espresso/espresso.py [hunk 2]
```diff
@@ -1516,11 +1517,12 @@ def write_input(self, inputname='pw.inp', calculation=None,
...
                 print('  {0:s}={1:s},'.format(attr, num2str(value)), file=finp)
 
-        sys_str_attrs = ['exxdiv_treatment']
+        sys_str_attrs = ['exxdiv_treatment','vdw_corr']
         for attr in sys_str_attrs:
             value = getattr(self, attr)
             if value is not None:
-                print('  {0:s}={1:s},'.format(attr, value), file=finp)
+                print("  {0:s}='{1:s}',".format(attr, value), file=finp)
+               # print('  {0:s}=\'{1:s}\','.format(attr, value), file=finp)
```
Comment (lmmentel) [inline espresso/espresso.py]: can you remove the commented line?

**Review by lmmentel — CHANGES_REQUESTED** [review]

File: espresso/espresso.py (hunk 2 above)
Comment (sandipde) [reply espresso/espresso.py]: done !

**Review by lmmentel — APPROVED** [review]
Great work!

**Comment by sandipde** [comment]
> Great work!

Thanks for mergeing ! makes life easier for all of us :-)

**Pull request merged**